pkg/asset/installconfig/platform.go

```diff
@@ -24,7 +24,9 @@ import (
 
 // Platform is an asset that queries the user for the platform on which to install
 // the cluster.
-type platform types.Platform
+type platform struct {
```

Comment:
Why do you need this change?

Reply:
allows to get the platform name without depending on more assets : https://github.com/openshift/installer/pull/1460/files?file-filters%5B%5D=.go#diff-4d231774d44727d779db320e7b6c189eR39

Reply:
> allows to get the platform name without depending on more assets

:confused: do you mind expanding on that?
in the current usage, why not call `platform.Name()` directly compared to `CurrentPlatformName` ?

Reply:
that gives returns the name of the asset. not the name of the platform.

Reply:
https://github.com/openshift/installer/blob/0be524e5a2d67459bc0ee6e2614b6b257c213de0/pkg/asset/installconfig/platform.go#L81